packages/blockchain-wallet-v4-frontend/src/data/preferences/reducers.js

```diff
@@ -36,6 +38,14 @@ const preferences = (state = INITIAL_STATE, action) => {
       const { displayed } = payload
       return assoc('showEtherWelcome', displayed, state)
     }
+    case AT.SET_BITCOIN_WELCOME: {
+      const { displayed } = payload
+      return assoc('showBitcoinWelcome', displayed, state)
```

Comment:
Looks good, nbd but try to use coin abbreviations instead of name (I know ether is used here as well).

The reason this works is because of persistStore which saves preferences to localStorage.